### PR TITLE
#331 Added feedback from users and enhancements

### PR DESCRIPTION
**Pull request by pavlas77**

### CODE REVIEW

**Comment by pavlas77** [comment]
@digitaldiina Not sure how to fix the link issue - space between TIP and CAUTION block.

File: docs/online/migrate/index.md [hunk 1]
```diff
@@ -139,10 +153,15 @@ You can choose from 2 options:
 
 ![22-recovery.png -screenshot][img18]
 
+## Migration log file
+
+A log file of the migration is written to %temp%\SoMigration folder.
```
Comment (digitaldiina) [inline docs/online/migrate/index.md:158]: Folder names in italics

File: docs/online/migrate/index.md [hunk 2]
```diff
@@ -47,13 +47,23 @@ The migrator needs access to the SuperOffice Admin client with a user with Super
...
 
 The SuperOffice CRM Online Migration Tool (OMT) is responsible for transferring a local onsite database and document archive to CRM Online. It also makes sure that a set of initial configuration steps is carried out.
 
+ OMT needs access to SuperOffice installation - database, so_arc folder, Service folders.
+
+> [!TIP]
+>
+> If you plan to use identify provider for the authentication, we recommend to [register][4] the customer domain name before doing the migration, so that users can immediately login using IDP and you don't need to do any user-transition later.
+
 > [!CAUTION]
```
Comment (digitaldiina) [inline docs/online/migrate/index.md:64]: Don't stack admonitions. Either change the tip or warning into normal para or insert a useful para between them (if you need to preserve the formatting).

File: docs/online/migrate/index.md [hunk 3]
```diff
@@ -47,13 +47,23 @@ The migrator needs access to the SuperOffice Admin client with a user with Super
...
+
 ## Start Online Migration Tool
 
 The SuperOffice CRM Online Migration Tool (OMT) is responsible for transferring a local onsite database and document archive to CRM Online. It also makes sure that a set of initial configuration steps is carried out.
 
+ OMT needs access to SuperOffice installation - database, so_arc folder, Service folders.
+
+> [!TIP]
+>
+> If you plan to use identify provider for the authentication, we recommend to [register][4] the customer domain name before doing the migration, so that users can immediately login using IDP and you don't need to do any user-transition later.
```
Comment (digitaldiina) [inline docs/online/migrate/index.md:62]: let the link text be "register the custom domain name"

**Review by digitaldiina — CHANGES_REQUESTED** [review]

**Pull request merged**